iina/Base.lproj/MainMenu.xib

```diff
@@ -264,12 +264,12 @@
                             <menuItem title="Volume:" enabled="NO" id="iQF-1S-9EK">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                             </menuItem>
-                            <menuItem title="Volume + 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
-                            <menuItem title="Volume + 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
+                            <menuItem title="Increase Volume 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
+                            <menuItem title="Increase Volume 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
```

Comment:
I meant the names of the touch bar items.
<img width="1102" alt="screen shot 2017-07-07 at 11 53 20 am" src="https://user-images.githubusercontent.com/20237141/27942626-e875a33e-630a-11e7-8d1b-b4e2b918a940.png">